rfc/updates.md

```diff
@@ -267,3 +267,28 @@ Upgrading Constellation microservices to 2.2.0:
 Warning: Please backup any important components before upgrading Kubernetes
 Apply change [yes/No]?
 ```
+
+# Compatibility
+`constellation-upgrade` has to handle the version of four components: CLI, image, microservices and Kubernetes.
+To do this correctly and keep the cluster in a working condition some constraints are required.
+
+- Microservice, image and CLI are versioned in lockstep, i.e. each release a new version of all componentes is released.
+  Each of these version numbers is one Constellation version.
+- Each Constellation version is compatible to three or four Kubernetes versions.
+  When a new Kubernetes version is released Constellation will support four versions for one release cycle; before phasing out the oldest Kubernetes version.
+  To learn if microserviceVersion A.B.C is compatible with Kubernetes version X.Y, one has to check whether the Constellation version is compatible with the Kubernetes version X.Y.
+  The Constellation version will be the same as the microserviceVersion: A.B.C. Same goes for the image and CLI.
+- Each Constellation version X.Y.Z is compatible with all patch versions of another Constellation version X.Y+1.
+- The individual versions of Microservice, image and CLI all have to be compatible with the targeted Kubernetes version.
+  Without loss of generality, this means that during a Kubernetes upgrade the oldest Constellation component has to be compatible with the new Kubernetes version.
```

Comment:
```suggestion
This means that during a Kubernetes upgrade the oldest Constellation component has to be compatible with the new Kubernetes version.
```
We should probably explicitly establish that one can upgrade e.g. only the microservices. Then it is clear what "**oldest** Constellation component" means. If this is what you are going for here.